dsf-bpe/dsf-bpe-server/src/main/java/org/highmed/dsf/fhir/subscription/AbstractExistingResourceLoader.java

```diff
@@ -10,17 +10,20 @@
 
 import javax.ws.rs.core.UriBuilder;
 
-import org.highmed.dsf.fhir.task.TaskHandler;
+import org.highmed.dsf.fhir.websocket.LastEventTimeIo;
+import org.highmed.dsf.fhir.websocket.ResourceHandler;
 import org.highmed.fhir.client.FhirWebserviceClient;
 import org.hl7.fhir.r4.model.Bundle;
 import org.hl7.fhir.r4.model.Bundle.BundleEntryComponent;
-import org.hl7.fhir.r4.model.Task;
+import org.hl7.fhir.r4.model.Resource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ExistingTaskLoader
+import ca.uhn.fhir.model.api.annotation.ResourceDef;
+
+public abstract class AbstractExistingResourceLoader<R extends Resource> implements ExistingResourceLoader<R>
```

Comment:
This class may not need to be abstract. The `ExistingQuestionnaireResponseLoader` and `ExistingTaskLoader` classes may not need to exist.